src/App.js

```diff
@@ -1,25 +1,101 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import styled from 'styled-components'
+import Header from './Components/Header'
+import MedicationGroup from './Components/MedicationGroup'
+
+export default function App() {
+  const currentDate = new Date()
+
+  const medication = [
+    {
+      id: 1,
+      time: '8:00',
+      meds: [
+        { id: 1, medName: 'Spironolacton' },
+        { id: 2, medName: 'Enalapril' },
+        { id: 3, medName: 'Prednisolon' },
+        { id: 4, medName: 'MMF' },
+        { id: 5, medName: 'Magnesium' },
+        { id: 6, medName: 'ASS' },
+      ],
+    },
+    { id: 2, time: '9:00', meds: [{ id: 1, medName: 'Tacrolimus' }] },
+    {
+      id: 3,
+      time: '13:00',
+      meds: [
+        { id: 1, medName: 'Magnesium' },
+        { id: 2, medName: 'MMF' },
+      ],
+    },
+    {
+      id: 4,
+      time: '18:00',
+      meds: [
+        { id: 1, medName: 'Magnesium' },
+        { id: 2, medName: 'MMF' },
+        { id: 3, medName: 'Enalapril' },
+        { id: 4, medName: 'Calcium' },
+      ],
+    },
+    { id: 5, time: '21:00', meds: [{ id: 1, medName: 'Tacrolimus' }] },
+  ]
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <Grid>
+      <Header>{formatDate(currentDate)}</Header>
+      <Flexbox>
+        {medication.map(({ id, time, meds }) => (
+          <MedicationGroup key={id} time={time} meds={meds} />
+        ))}
+      </Flexbox>
+    </Grid>
+  )
+
+  function formatDate(date) {
```

Comment:
In a future PR or user story this could be in it's own component, something like `DateHeader` or something like that? The App.js should always be as short and clean as possible.